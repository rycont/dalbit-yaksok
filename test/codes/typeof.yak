선언한_변수 = 10
선언한_변수 의 값 종류 보여주기

두번째_변수 = "네"
두번째_변수 의 값 종류 보여주기