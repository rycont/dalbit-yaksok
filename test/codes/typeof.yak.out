숫자
문자
